engine/install/binaries.md

```diff
@@ -120,13 +133,21 @@ instructions for enabling and configuring AppArmor or SELinux.
 
 ## Install client binaries on macOS
 
-The macOS binary includes the Docker client only. It does not include the
-`dockerd` daemon.
+> **Note**
+>
+> The following instructions are mostly suitable for testing purposes. The macOS binary includes the Docker client only. It does not include the
+`dockerd` daemon which is required to run containers. Therefore, we recommend that you install [Docker Desktop](../../desktop/index.md) instead.
+
+The binaries for Mac also do not contain:
+
+-  A runtime environment. You must set up a functional engine either in a VM, or on a remote Linux machine.
```

Comment:
Double space here; perhaps "Virtual Machine" instead of VM?

```suggestion
- A runtime environment. You must set up a functional engine either in a Virtual Machine, or on a remote Linux machine.
```